src/main/java/com/information/user/UserService.java

```diff
@@ -0,0 +1,22 @@
+package com.information.user;
+
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+public class UserService {
+    private final UserMapper userMapper;
+
+    public UserService(UserMapper userMapper) {
+        this.userMapper = userMapper;
+    }
+
+    public List<User> findAll() {
+        return userMapper.getAll();
+    }
+
+    public List<User> findById(int id) {
+        return userMapper.getId(id);
```

Comment:
ID を get するのではなく ID で get するので getById がよいかとおもいます。
```suggestion
        return userMapper.getById(id);
```

Reply:
おっしゃるっ通りです。
意味を理解した上で適切に命名できるよう心がけたいと思います。

修正しましたのでご確認お願いします。
f819c53dcdba8cde541cb9f0e1acbb3923e8ff63